certbot/certbot/_internal/cli/cli_constants.py

```diff
@@ -43,6 +43,7 @@
     register        Create an ACME account
     unregister      Deactivate an ACME account
     update_account  Update an ACME account
+    fetch_account   Fetch ACME account data from server and print the contact info
```

Comment:
What do you think about naming the verb something else?

The current name is accurate, but to me it feels like it is exposing the guts of Certbot+ACME a bit. Like, does the user really need to know that a server round-trip is required to get this information? Is it enough to just tell the user this will show their account information?

```suggestion
    show_account   Display details of the registered ACME account
```

Reply:
The name of the verb was just a first "draft" if you will, a more generic name would be more accurate now we're including more info indeed.